docs/marketplace/dev/feature-integration-guides/202204.0/marketplace-merchant-custom-prices.md

```diff
@@ -0,0 +1,232 @@
+
+{% info_block warningBox "Verification" %}
+
+Log in to the Merchant Portal with a merchant that has at least one merchant relationship.
+
+Open any merchant product and make sure that the Prises table contains the "Customer" column for both: abstract and concrete products.
+Make sure that you can filter and sort the price table by Customer column.
+
+{% endinfo_block %}
+
+<details>
+<summary markdown='span'>src/Pyz/Service/PriceProduct/PriceProductDependencyProvider.php</summary>
+
+```php
+<?php
+
+namespace Pyz\Service\PriceProduct;
+
+use Spryker\Service\PriceProduct\PriceProductDependencyProvider as SprykerPriceProductDependencyProvider;
+use Spryker\Service\PriceProductMerchantRelationship\Plugin\PriceProduct\MerchantRelationshipPreBuildPriceProductGroupKeyPlugin;
+
+class PriceProductDependencyProvider extends SprykerPriceProductDependencyProvider
+{
+    /**
+     * @return array<int, \Spryker\Service\PriceProductExtension\Dependency\Plugin\PreBuildPriceProductGroupKeyPluginInterface>
+     */
+    protected function getPreBuildPriceProductGroupKeyPlugins(): array
+    {
+        return [
+            new MerchantRelationshipPreBuildPriceProductGroupKeyPlugin(),
+        ];
+    }
+}
+```
+
+</details>
+
+{% info_block warningBox "Verification" %}
+
+Open any merchant product with a regular price.
+Create a customer-specific price with the same combination of currency and country as the existing price.
+Make sure that there is no validation error.
+
+{% endinfo_block %}
+
+<details>
+<summary markdown='span'>src/Pyz/Zed/PriceProduct/PriceProductDependencyProvider.php</summary>
+
+```php
+<?php
+
+namespace Pyz\Zed\PriceProduct;
+
+use Spryker\Zed\PriceProduct\PriceProductDependencyProvider as SprykerPriceProductDependencyProvider;
+use Spryker\Zed\PriceProductMerchantRelationship\Communication\Plugin\PriceProduct\MerchantRelationshipPriceProductCollectionDeletePlugin;
+use Spryker\Zed\PriceProductMerchantRelationshipMerchantPortalGui\Communication\Plugin\PriceProduct\MerchantRelationshipVolumePriceProductValidatorPlugin;
+
+class PriceProductDependencyProvider extends SprykerPriceProductDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\PriceProductExtension\Dependency\Plugin\PriceProductValidatorPluginInterface>
+     */
+    protected function getPriceProductValidatorPlugins(): array
+    {
+        return [
+            new MerchantRelationshipVolumePriceProductValidatorPlugin(),
+        ];
+    }
+
+    /**
+     * @return array<\Spryker\Zed\PriceProductExtension\Dependency\Plugin\PriceProductCollectionDeletePluginInterface>
+     */
+    protected function getPriceProductCollectionDeletePlugins(): array
+    {
+        return [
+            new MerchantRelationshipPriceProductCollectionDeletePlugin(),
+        ];
+    }     
+}
+```
+
+</details>
+
+{% info_block warningBox "Verification" %}
+
+Make sure that you see the validation error while attempting to set or create the customer price for the volume price.
```

Comment:
can we rephrase it to say customer-specific prices don't include/ support volume prices / cannot be combined with volume prices